azure-functions-codegen/src/func.rs

```diff
@@ -375,49 +415,48 @@ fn bind_output_type(
 fn bind_return_type(
     ret: &ReturnType,
     binding_args: &mut HashMap<String, (AttributeArgs, Span)>,
+    is_activity: bool,
```

Comment:
This is my Python speaking, but if Rust had support for defaults (without being so verbose), it would be nice to default this to `false` just because activity functions are a "non-normal" case.

Reply:
Agreed, it sure would be nice if we had default parameter values :)